File: Iniciantes/1048.py
def imprime(salario, reajuste, percentual):
	print "Novo salario: %.2f" % salario
	print "Reajuste ganho: %.2f" % reajuste
	print "Em percentual: %d %% " % percentual

salario = float(input())
tmp = 0

if salario <= 400.00 :
	tmp = salario * 0.15
	imprime(salario+tmp,tmp,15)
elif salario > 400.00 and salario <= 800.00 :
	tmp = salario * 0.12
	imprime(salario+tmp,tmp,12)	
elif salario > 800.00 and salario <= 1200.00 :
	tmp = salario * 0.10
	imprime(salario+tmp,tmp,10)	
elif salario > 1200.00 and salario <= 2000.00 :
	tmp = salario * 0.07
	imprime(salario+tmp,tmp,7)	
elif salario > 2000.00 :
	tmp = salario * 0.04
	imprime(salario+tmp,tmp,4)	

File: Iniciantes/1048.c
#include <stdio.h>

int main(void) {
	float salario, reaj;
	int perc;
	scanf("%f", &salario);

	if (salario <= 400){
		reaj = salario * 0.15;
		perc = 15;
	}else {
		if (salario > 400 && salario <= 800) {
			reaj = salario * 0.12;
			perc = 12;
			} else {
				if (salario > 800 && salario <= 1200){
					reaj = salario * 0.10;
					perc = 10;
					} else {
						if (salario > 1200 && salario <= 2000) {
							reaj = salario * 0.07;
							perc = 7;
						} else {
							if (salario > 2000){
								reaj = salario * 0.04;
								perc = 4;
								}
							}
						}
					}
		}	

	printf("Novo salario: %.2f\nReajuste ganho: %.2f\nEm percentual: %d %%\n", (salario+reaj), reaj, perc);

	return 0;
}